test/robot/record/shelf.go

```diff
@@ -44,11 +45,18 @@ func NewShelf(ctx context.Context, shelfURL string) (Shelf, error) {
 	switch location.Scheme {
 	case "", "file":
 		if location.Host != "" {
-			return nil, log.Err(ctx, nil, "Host not supported for file shelves")
+			return nil, log.Errf(ctx, nil, "Host not supported for file shelves")
 		}
 		if location.Path == "" {
 			return nil, log.Err(ctx, nil, "Path must be specified for file shelves")
 		}
+		if runtime.GOOS == "windows" {
+			if l := len(location.Path); l > 2 && location.Path[0] == '/' && location.Path[2] == ':' {
```

Comment:
If this is really needed, you can use [strings.TrimPrefix](https://golang.org/pkg/strings/#TrimPrefix)